### PR TITLE
Drop support for Scala 2.13

### PR DESCRIPTION
**Pull request by bcarter97**
## Related Issues

- Connects to #9

## Definition of Done:

The Below tasks should be completed before marking a PR as ready for review.

- [ ] Added / Removed / Updated relevant tests
- [ ] Scaladoc updated

### CODE REVIEW

File: src/test/scala/utils/EmbeddedKafka.scala [hunk 1]
```diff
@@ -16,39 +16,44 @@ trait EmbeddedKafka[F[_]] {
...
 
-  def createCustomTopic(topic: String, partitions: Int, topicConfig: Map[String, String])(implicit
+  def createCustomTopic(topic: String, partitions: Int, topicConfig: Map[String, String])(using
       kafkaConfig: EmbeddedKafkaConfig,
       F: Async[F]
   ): F[NonEmptyList[TopicPartition]] =
     for {
-      maybeCreate <-
-        F.delay(Underlying.createCustomTopic(topic = topic, topicConfig = topicConfig, partitions = partitions))
+      tpIdxs      <- F.fromOption(
```
Comment (lacarvalho91) [inline src/test/scala/utils/EmbeddedKafka.scala:27]: minor but this name doesn't mean much to me, whats the x?

**Review by lacarvalho91 — APPROVED** [review]
minor comment but otherwise LGTM

File: src/test/scala/utils/EmbeddedKafka.scala (hunk 1 above)
Comment (bcarter97) [reply src/test/scala/utils/EmbeddedKafka.scala:27]: Just an abbreviation of index

Comment (bcarter97) [reply src/test/scala/utils/EmbeddedKafka.scala]: [`7064f7f` (#17)](https://github.com/sky-uk/fs2-kafka-topic-loader/pull/17/commits/7064f7ff0ffc1658d6c3ef5c76e652d1eb1f5f8f)

Comment (lacarvalho91) [reply src/test/scala/utils/EmbeddedKafka.scala]: ah yeah of course, that didn't register for some reason

**Pull request merged**